src/Topics/BidsOneNotrump.hs

```diff
@@ -0,0 +1,132 @@
+module Topics.BidsOneNotrump(
+    b1N  -- Copied from StandardOpenings
+  , b1N2C
+  , b1N2D
+  , b1N2H
+  , b1N4D
+  , b1N4D4H
+  , b1N4H
+  , b1N4H4S
+  -- Strengths
+  , gameForcing  -- Game or stronger
+  , slamInterest
+  , slamInvite
+  , gameNoSlam
+  , invitational
+  , lessThanInvitational
+) where
+
+
+import Auction(forbid, pointRange, suitLength, minSuitLength, maxSuitLength,
+               Action, makeCall, makeAlertableCall, alternatives, longerThan)
+import Output((.+))
+import StandardOpenings(b1N)
+import qualified Terminology as T
+
+------------------
+-- Point ranges --
+------------------
+-- Parameterize the strength so this is easy to adapt for other notrump ranges
+minNotrumpStrength :: Int
+minNotrumpStrength = 15
+maxNotrumpStrength :: Int
+maxNotrumpStrength = 17
+strengthNeededForGame :: Int
+strengthNeededForGame = 25
+strengthNeededForSlam :: Int
+strengthNeededForSlam = 30
```

Comment:
Is this value right? I'm unsure. 33 HCP guarantees you're not missing 2 aces, but slam investigation conventions (e.g., bid RKC) are only useful if you might be missing 2 aces, so maybe 30 HCP is right after all? I dunno; I fiddled with the number until the hands seemed sufficiently slammy, but I could be way off on this. It would be neat to take some of the numeric modelling stuff tgwnn has described and investigate how much strength you really need for slam (or, ideally, just find someone else who has already done this and read what they found).

Reply:
:shrug: It's good enough for now, and can be changed later. Put in a todo to consider more.